fecgov/openFEC
# create new materialized view to include form_3 submitted by pac and party

## Summary 

Addresses [#3103](https://github.com/fecgov/openFEC/issues/3103) 

-V0081 was added to data/migration.  

## How to test the changes locally

 - If local database has enough sample data, the materialized view should have this row:
FILE_NUM | CMTE_ID | FORM_TP | RPT_TP | RPT_YR | BEGIN_IMAGE_NUM | 
1206344 | C00156554 | F3 | YE | 2017 | 201801240300187403

- otherwise, run a select statement against the query part only
